substrate/bin/node/cli/Cargo.toml

```diff
@@ -168,16 +169,16 @@ clap_complete = { optional = true, workspace = true }
 
 node-inspect = { optional = true, workspace = true, default-features = true }
 
-polkadot-sdk = { features = ["frame-benchmarking-cli", "sc-cli", "sc-storage-monitor", "substrate-build-script-utils"], optional = true, workspace = true, default-features = true }
+polkadot-sdk = { features = [
+	"frame-benchmarking-cli",
+	"sc-cli",
+	"sc-storage-monitor",
+	"substrate-build-script-utils",
+], optional = true, workspace = true, default-features = true }
```

Comment:
This is strange, running taplo fmt configured as the repo requires will not format this version of  Cargo.toml, and will not format it on master either, where it is a single line. So both are conforming with our formatting rules. I am not against this change (might be easier to follow on smaller sized screens that are not wrapping around), but yeah, now I am actually thinking if the rest of the changes that are kept are conforming to the repo formatting rules. Can you double check please?

Reply:
> This is strange, running taplo fmt configured as the repo requires will not format this version of Cargo.toml, and will not format it on master either, where it is a single line. So both are conforming with our formatting rules. I am not against this change (might be easier to follow on smaller sized screens that are not wrapping around), but yeah, now I am actually thinking if the rest of the changes that are kept are conforming to the repo formatting rules. Can you double check please?

Maybe I am wrong. After I disable my extension, it just can keep my change, not like before, once I save file it will restore. I remember that I already commit many times , on those commit, the toml are also many lines, but ci never report error(our ci includes toml format check). So maybe both are conforming with our formatting rules. Or we need to modify taplo.toml, I guess current rules don't work on new rust because we bump ci rust to 1.81 recently.

Reply:
I even found some single-line becomes multi-lines after run this command.

Reply:
Which command? This: `env TAPLO_CONFIG=.config/taplo.toml taplo fmt cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/Cargo.toml`?

Reply:
> Which command? This: `env TAPLO_CONFIG=.config/taplo.toml taplo fmt cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/Cargo.toml`?

`taplo format --config .config/taplo.toml`

Reply:
For me `taplo format --config .config/taplo.toml` (run from the root of repo) does not change anything.
My version:
```
taplo --version
taplo 0.9.0
```

Reply:
It seems that now formatting is correct accoring to CI check:
https://github.com/paritytech/polkadot-sdk/actions/runs/11368634327/job/31624406299?pr=5804

Reply:
> It seems that now formatting is correct according to CI check: https://github.com/paritytech/polkadot-sdk/actions/runs/11368634327/job/31624406299?pr=5804

Yeah, however, formatting check in my previous commit is also correct. So multi-lines is acceptable?

Reply:
If CI says it is fine, then probably yes.
I'd personally prefer consistent formatting, but you can leave it as is. Up to you.

Reply:
> If CI says it is fine, then probably yes. I'd personally prefer consistent formatting, but you can leave it as is. Up to you.

I'd also personally prefer consistent formatting. I think this is CI bug.

Reply:
Not really a CI bug, but some formatting rules which are not restrictive enough. I wouldn't invest time in enforcing singe/multi-line rules either, so resolving this.